src/Compilers/Core/Portable/Diagnostic/Diagnostic.cs

```diff
@@ -220,7 +220,7 @@ public abstract partial class Diagnostic : IEquatable<Diagnostic?>, IFormattable
         /// <param name="severity">The diagnostic's effective severity.</param>
         /// <param name="defaultSeverity">The diagnostic's default severity.</param>
         /// <param name="isEnabledByDefault">True if the diagnostic is enabled by default</param>
-        /// <param name="warningLevel">The warning level, between 1 and 4 if severity is <see cref="DiagnosticSeverity.Warning"/>; otherwise 0.</param>
+        /// <param name="warningLevel">The warning level, greater than 1 if severity is <see cref="DiagnosticSeverity.Warning"/>; otherwise 0.</param>
```

Comment:
```suggestion
        /// <param name="warningLevel">The warning level, greater than 0 if severity is <see cref="DiagnosticSeverity.Warning"/>; otherwise 0.</param>
```